pysiaf/utils/tools2.py

```diff
@@ -0,0 +1,217 @@
+"""A collection of helper functions to support pysiaf calculations and functionalities
+
+Authors
+-------
+
+    - Colin Cox
+
+References
+----------
+
+
+
+"""
+
+import sys
+import numpy as np
+from math import *
```

Comment:
Please either resort to using numpy functions only (e.g. np.deg2rad, np.cos), which I recommend, or import the math functions using `import math`

Reply:
Please do not use 
`from math import *`